Doc/using/configure.rst

```diff
@@ -804,11 +804,14 @@ Libraries options
 
 .. option:: --with-system-libmpdec
 
-   Build the ``_decimal`` extension module using an installed ``mpdec``
-   library, see the :mod:`decimal` module (default is no).
+   Build the ``_decimal`` extension module using an installed ``mpdecimal``
+   library, see the :mod:`decimal` module (default is yes).
 
    .. versionadded:: 3.3
 
+   .. versionchanged:: 3.13
+      Default to using the installed ``mpdecimal`` library.
+
```

Comment:
We could also `.. deprecated-removed:: 3.13 3.15` this option.
```suggestion

   .. deprecated-removed:: 3.13 3.15
      A copy of the ``mpdecimal`` library sources will no longer be distributed
      with Python 3.15.

```

Iffy on the exact wording :)

Reply:
Yeah, this is probably a good idea :) I don't have a better suggestion, so let's go with this for now.